src/components/custom-field-input-multiple-choice/custom-field-input-multiple-choice.test.jsx

```diff
@@ -270,4 +271,79 @@ describe('<CustomFieldInputMultipleChoice>', () => {
       expect(screen.getByText('Choice 2')).toBeInTheDocument();
     });
   });
+
+  describe('dropdown close behavior', () => {
+    it('closes the dropdown when clicking outside', async () => {
+      const choices = [{
+        id: 'broke',
+        label: 'broke my heart',
+      }, {
+        id: 'now',
+        label: "now I'm aching for you",
+      }];
+
+      render(
+        <div>
+          <span>CLOSE</span>
+          <CustomFieldInputMultipleChoice {...requiredProps} choices={choices} />
+        </div>,
+      );
+
+      userEvent.click(screen.getAllByLabelText('test label')[0]);
```

Comment:
We prefer to not use `getAll*`

```suggestion
      userEvent.click(screen.getByLabelText('test label', { selector: 'input' ));
```